python/caffe/test/test_net.py

```diff
@@ -2,6 +2,7 @@
 import tempfile
 import os
 import numpy as np
+import six
```

Comment:
Oh right, thanks for the reminder.

On Mon, Jul 20, 2015 at 21:33 Philipp Krähenbühl <notifications@github.com>
wrote:

> In python/caffe/test/test_net.py
> <https://github.com/BVLC/caffe/pull/2740#discussion_r35069069>:
>
> > @@ -2,6 +2,7 @@
> >  import tempfile
> >  import os
> >  import numpy as np
> > +import six
>
> already done in #2714 <https://github.com/BVLC/caffe/pull/2714>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BVLC/caffe/pull/2740/files#r35069069>.
>